packages/ui/src/views/user-management/components/dialogs/components/remove-admin/remove-admin.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, ButtonGroup, Dialog } from '@/components'
+import { IRemoveAdminDialogProps } from '@/views/user-management/components/dialogs/components/remove-admin/types'
+import { useStates } from '@/views/user-management/providers/state-provider'
+import { useUserManagementStore } from '@/views/user-management/providers/store-provider'
+
+export function RemoveAdminDialog({ handleUpdateUserAdmin, open, onClose }: IRemoveAdminDialogProps) {
+  const { useTranslationStore, useAdminListUsersStore } = useUserManagementStore()
+  const { t } = useTranslationStore()
+  const { user } = useAdminListUsersStore()
+
+  const { loadingStates, errorStates } = useStates()
+  const { isUpdatingUserAdmin } = loadingStates
+  const { updateUserAdminError } = errorStates
+
+  const isAdmin = user?.admin ?? false
+
+  const handleSubmit = () => {
+    handleUpdateUserAdmin(user?.uid || '', !isAdmin)
+  }
+
+  return (
+    <Dialog.Root open={open} onOpenChange={onClose}>
+      <Dialog.Content className="max-w-xl">
+        <Dialog.Header>
+          <Dialog.Title className="font-medium">
+            {t('views:userManagement.updateAdminRights', 'Are you sure you want to remove {name} as an admin?', {
+              name: user?.display_name
+            })}
+          </Dialog.Title>
+        </Dialog.Header>
+
+        {/* TODO: investigate how to make user login bold with font-weight: 600 */}
```

Comment:
Not quite sure what "user login" is referring to 🤔

Reply:
You can use `dangerouslySetInnerHTML` to do that https://github.com/harness/canary/pull/1131#discussion_r1985851438